index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>Document</title>
+</head>
+
+<body>
+Dec 20th, 10.30 AM EST (Zoom)
```

Comment:
You have the content all sorted here, but not the expected markup -- There should be more HTML providing meaning and structure to this content. Let me know if you want more specific feedback or other examples to look at?